webserver/cWebem.cpp

```diff
@@ -1886,16 +1804,16 @@ namespace http {
 
 		bool cWebemRequestHandler::CheckAuthentication(WebEmSession & session, const request& req, reply& rep)
 		{
-			bool bTrustedNetwork = false;
-
 			session.rights = -1; // no rights
 			session.id = "";
 			session.username = "";
 			session.auth_token = "";
+			session.istrustednetwork = false;
```

Comment:
No need to initialize this to false as this is done in the constructor of the object, same for the above strings